frontend/packages/dev-console/src/components/import/route/CreateRoute.tsx

```diff
@@ -1,26 +1,17 @@
 import * as React from 'react';
-import * as _ from 'lodash';
 import { useFormikContext, FormikValues } from 'formik';
 import { TextInputTypes } from '@patternfly/react-core';
-import { InputField, DropdownField } from '@console/shared';
-import { makePortName } from '../../../utils/imagestream-utils';
+import { InputField } from '@console/shared';
+import RouteInputField from './RouteInputField';
 
 const CreateRoute: React.FC = () => {
   const {
     values: {
       image: { ports },
-      route: { defaultUnknownPort, targetPort },
+      route: { defaultUnknownPort },
     },
   } = useFormikContext<FormikValues>();
```

Comment:
```suggestion
  } = useFormikContext<DeployImageFormData | GitImportFormData>();
```

Adding this types so that the ports and defaultUnknownPort vars are typed.

Reply:
Updated, didn't know we could do that. Now I'm wondering why we even use `FormikValues` type :P.